corehq/ex-submodules/casexml/apps/case/mock/case_block.py

```diff
@@ -35,8 +35,7 @@ def __init__(self, case_id, date_modified=None, user_id=undefined,
         self.update = copy.copy(update) if update else {}
         now = datetime.utcnow()
         self.date_modified = date_modified or now
-        self.date_opened = (now if create and date_opened is CaseBlock.undefined
-                            else date_opened)
+        self.date_opened = date_opened
```

Comment:
"date_opened automatically is set to date_modified"
Hey @dannyroberts , where does this happen?

Reply:
Hey sorry missed this yesterday after the diff became outdated. Answered it later here when Simon asked it again: https://github.com/dimagi/commcare-hq/pull/27875#discussion_r444253403

Reply:
Cool, thanks @dannyroberts